worldedit_commands/init.lua

```diff
@@ -12,8 +12,13 @@ worldedit.prob_list = {}
 
 local safe_region, reset_pending = dofile(minetest.get_modpath("worldedit_commands") .. "/safe.lua")
 
+-- The prefix avoids chat messages being mixed up with WorldEdit messages
+worldedit.notify_form = minetest.get_color_escape_sequence("#94ffcc") ..
+	"WorldEdit -!- %s" .. minetest.get_color_escape_sequence("#ffffff")
 function worldedit.player_notify(name, message)
-	minetest.chat_send_player(name, "WorldEdit -!- " .. message, false)
+	-- FIXME: What is the third chat_send_player parameter?

```

Comment:
A long time ago, it meant that the "Server -!-" prefix should not be added to message.